### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by agracey**
Borrowed from Rancher Desktop

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+# Contributing to Epinio
+
+Rancher Desktop accepts contributions via GitHub issues and pull requests.
```
Comment (andreas-kupries) [inline CONTRIBUTING.md:3]:
```suggestion
Epinio accepts contributions via GitHub issues and pull requests.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+This document outlines the process to get your pull request accepted.
+
+## Start With An Issue
+
+Prior to creating a pull request it is a good idea to [create an issue].
+This is especially true if the change request is something large.
+The bug, feature request, or other type of issue can be discussed prior to
+creating the pull request. This can reduce rework.
+
+[create an issue]: https://github.com/rancher-sandbox/rancher-desktop/issues/new
```
Comment (andreas-kupries) [inline CONTRIBUTING.md:13]:
```suggestion
[Create an issue]: https://github.com/epinio/epinio/issues/new
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+## Sign Your Commits
+
+A sign-off is a line at the end of the explanation for a commit.
+All commits must to be signed. Your signature certifies that you wrote the patch
```
Comment (andreas-kupries) [inline CONTRIBUTING.md:18]:
```suggestion
All commits have to be signed. Your signature certifies that you wrote the patch
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+    this project or the open source license(s) involved.
+```
+
+Then you add a line to every git commit message:
+
+    Signed-off-by: Joe Smith <joe.smith@example.com>
+
+Use your real name (sorry, no pseudonyms or anonymous contributions).
+
+If you set your `user.name` and `user.email` git configs, you can sign your
```
Comment (andreas-kupries) [inline CONTRIBUTING.md:68]:
```suggestion
If you set your `user.name` and `user.email` in your local git configuration, you can sign your
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+```
+Author: John Smith <john.smith@example.com>
+Date:   Thu Feb 2 11:41:15 2018 -0800
+
+    Update README
+
+    Signed-off-by: John Smith <john.smith@example.com>
+```
+
+Notice the `Author` and `Signed-off-by` lines match. If they don't your PR will
```
Comment (andreas-kupries) [inline CONTRIBUTING.md:83]:
```suggestion
Notice how the `Author` and `Signed-off-by` lines match. If they don't your PR will
```

Question: Does epinio have this automated DC0 check ?

**Review by andreas-kupries — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (agracey) [reply CONTRIBUTING.md:83]: Can we commit with this language and add the DC0 check next week? I'm asking Matt how they set this up.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**